index.js

```diff
@@ -206,7 +206,10 @@
 
   'use strict';
 
-  var sentinel = {};
+  /* istanbul ignore next */
+  var global_ =
+  typeof self === 'object' && self != null && self.self === self ? self
+                                                                 : global;
```

Comment:
I never feel good writing this sort of code. It looks error-prone. Let me know if you think there's a better approach.